src/Raven.Client/ServerWide/Operations/EditRevisionsForConflictsOperationResult.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Raven.Client.ServerWide.Operations
+{
+    public class EditRevisionsForConflictsOperationResult
```

Comment:
Rename to `ConfigureRevisionsForConflictsOperationResult`?

Reply:
done